### DISCUSSION TITLE
Not getting verification email in my college email.

### DISCUSSION THREAD

**Discussion by ravisharmaa** [community/community#31563] [GitHub Education]
I wanted to add college email in for education pack but did not get any email for verification need help.

**Comment by toufiqul-hasan** [comment]
To be eligible for GitHub Global Campus, including GitHub Student Developer Pack and other benefits, you must:

1.Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
2.Have a verifiable school-issued email address or upload documents that prove your current student status
3.Have a GitHub personal account
4.Be at least 13 years old

Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

During your tenure as a student, you may be prompted to periodically re-verify your current academic status.

**Reply by ravisharmaa** [reply]
Yes I have the educational institute email, I have a personal account as well, but unable to receive an email for verification.

**Reply by toufiqul-hasan** [reply]
Then you need to submit a valid student identification card, or other official proof of enrollment.
You can also provide your college bonafide or any other document which mentions that you are enrolled.

Apply process:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student
